conans/client/conf/detect_vs.py

```diff
@@ -151,7 +151,7 @@ def vswhere(all_=False, prerelease=False, products=None, requires=None, version=
     try:
         from conans.util.runners import check_output_runner
         cmd = cmd_args_to_string(arguments)
-        output = check_output_runner(cmd).strip()
+        output = check_output_runner(cmd, encoding="ansi").strip()
```

Comment:
I am a bit concerned this could break to some users with different system encodings, like Chinese, Japanese, etc, because this doesn't mandate the output encoding, just used to ``load()`` the resulting file, that might not be really "ansi" always, but other things and crash with an EncodeError?


https://devblogs.microsoft.com/setup/vswhere-now-supports-utf8-to-override-console-code-page/ was the introduction of the utf8 flag you commented, so it seems old enough to trust it?

Reply:
Yes, I tested it locally and it also works. Maybe this is the more defensive and safer approach.